science/orthanc/Makefile

```diff
@@ -0,0 +1,57 @@
+# Created by: maintainer.freebsd@xpoundit.com
```

Comment:
I somehow don't know if we should keep this created by line before it was deleted

https://svnweb.freebsd.org/ports/head/science/orthanc/Makefile?revision=471770&view=markup&pathrev=471893#l1

Or this is also fine.

I'm OK for both, maybe the new maintainer revives this is better

Reply:
I have no strong feelings, but slightly prefer the current maintainer, so I will allow it ;)